nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/vision/StartGcpVisionAnnotateFilesOperation.java

```diff
@@ -56,12 +56,12 @@ public class StartGcpVisionAnnotateFilesOperation extends AbstractStartGcpVision
                     "                \"mimeType\": \"application/pdf\"\n" +
                     "            },\n" +
                     "            \"features\": [{\n" +
-                    "                    \"type\": \"DOCUMENT_TEXT_DETECTION\",\n" +
+                    "                    \"type\": \"${feature-type}\",\n" +
```

Comment:
This needs to be adjusted to `vision-feature-type`:
```suggestion
                    "                    \"type\": \"${vision-feature-type}\",\n" +
```

Reply:
Thanks fixed.